src/StackHCI/help/Register-AzStackHCI.md

```diff
@@ -94,7 +95,37 @@ Parameter Sets: (All)
 Aliases:
 
 Required: False
-Position: 9
+Position: 10
```

Comment:
The order of positional parameters cannot be changed. That was a breaking change. Append new parameters at the end of the parameter list.

Reply:
Fixed. thanks !